### ISSUE TITLE
About CTC loss!

### ISSUE THREAD

**Issue by thunder123321** [rosinality/imputer-pytorch#1]
I'd like to ask you something. Can I calculate loss with cross entropy loss on the generated ctc_align sequence instead of torch.nn.CTCloss() in the code？
